src/features/auth/pages/signup/AfterSignupEmailValidationBuffer/AfterSignupEmailValidationBuffer.tsx

```diff
@@ -32,14 +29,11 @@ export function AfterSignupEmailValidationBuffer() {
   const { params } = useRoute<UseRouteType<'AfterSignupEmailValidationBuffer'>>()
 
   useEffect(() => {
-    // Wait for feature flag state to be fetched from Firestore
-    if (enableTrustedDevice === undefined) return
-    // Wait for device info when feature flag is active
-    if (enableTrustedDevice && !deviceInfo) return
+    if (!deviceInfo) return
 
     beforeEmailValidation()
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [enableTrustedDevice, enableTrustedDevice && !deviceInfo])
+  }, [!deviceInfo])
```

Comment:
C'est pas ça plutôt ? 
```suggestion
  }, [deviceInfo])
```